.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "trailingComma": "all",
+  "singleQuote": true,
+  "semi": true,
+  "tabWidth": 2,
+  "printWidth": 80
```

Comment:
I would suggest to increase the `printWidth` to around 140.

Reply:
General wrap lengths are 80, 120, but some use 150. I prefer 120 :D

Reply:
I got used to 160 with the SDK :D
maybe **`127.6²`** ? (:

Reply:
I got used to 160 with the SDK :D
maybe **`127.6²`** ? (: